tutorials/assets_pipeline/escn_exporter/index.rst

```diff
@@ -3,48 +3,12 @@
 Blender ESCN exporter
 =====================
 
-.. note:: This chapter relates to the Blender plugin called "Godot Blender Exporter",
-          which can be downloaded here: https://github.com/godotengine/godot-blender-exporter
+To export from Blender to Godot 4.x, use one of the 
+:ref:`available 3D formats <doc_importing_3d_scenes_available_formats>`.
 
-This plugin can be used to export Blender scenes in a Godot-specific scene format
-called ESCN, which is similar to TSCN (text format) but will be imported as binary
-SCN for performance.
+The plugin `Godot Blender Exporter <https://github.com/godotengine/godot-blender-exporter>`__ 
+is not maintained or supported in Godot 4.x. While not officially supported, the plugin may 
+partially work for some Godot and Blender versions, particularly before Blender version 4.0.
+For complete docs on the Blender exporter, see the 
+`previous version of this page <https://docs.godotengine.org/en/4.0/tutorials/assets_pipeline/escn_exporter/index.html>`__.
```

Comment:
This should link to the last docs version that keeps the full set of pages. That will be either `3.6`, or maybe `4.0` or `4.1` if we only backport this PR as far as `4.2`.

Alternately, we could not link to the previous docs version at all, since that is implicitly supported by RTD version switching. I thought it would be good to leave an explicit link here, in case someone finds their way to this page or is still relying on the partial functionality of the addon.

Reply:
I looked through the git blame of the [various pages](https://github.com/godotengine/godot-docs/tree/f6d53d141edad708f2106f77723895c66c65b674/tutorials/assets_pipeline/escn_exporter) changed or removed in this PR, and besides unrelated naming or spelling changes, the last change was 4 years ago. 

So this removed version has no relevant changes compared to whichever version continues to exist, either `4.0`/`4.1` or `3.6`.